docs/config_ovos_audio.md

```diff
@@ -0,0 +1,242 @@
+## Audio Service
+
+The audio service is responsible for loading TTS and Audio plugins
+
+All audio playback is handled by this service
+
+
+### Native playback
+
+Usually playback is triggered by some originating bus message, eg `"recognizer_loop:utterance"`, this message contains metadata that is used to determine if playback should happen.
```

Comment:
```suggestion
Usually playback is triggered by some originating bus message, eg `"recognizer_loop:utterance"`. This message contains metadata that is used to determine if playback should happen.
```

Reply:
This feels very technical for this section given the rewrite.

Reply:
not used file for now